app.py

```diff
@@ -1,21 +1,32 @@
-import os
-from flask import Flask , request , render_template , send_file
-from flask import Response
+from flask import Flask , request , render_template , send_file,make_response,jsonify
 from tinydb import Query , TinyDB
-import io,json
+import io,json,os
 import pic_email as wc
-
+import jobs as jobs
+import traceback
 app = Flask(__name__)
+defaultResult_0="KEY is not right"
+defaultResult_1="KEY is right"
+defaultResult_2="Make Image& Send email sucess"
+defaultResult_3="Have been used"
+defaultResult_5="Send wrong"
+defaultResult_6="Sending Notice Email"
 # 0:KEY is not right
 # 1:KEY is right
 # 2: Make Image& Send email sucess
 # 3: Have been used
 # 4:
 # 5:Send wrong
-def return_msg(message):
-    if type(message) is dict:
-        message = json.dumps(message)
-    return message
+def errorHanddler(e=Exception,isOut=True):
+    print(str(e))
```

Comment:
all print lines of debug information should be deleted, or use some `verbose` technique to switch it on or off.